NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -458,6 +458,52 @@ TransferNCGHistory ToTransferNCGHistory(TxSuccess txSuccess, string? memo)
                 name: "addressQuery",
                 description: "Query to get derived address.",
                 resolve: context => new AddressQuery(standaloneContext));
+            Field<NonNullGraphType<BooleanGraphType>>(
+                name: "replayBlocks",
+                arguments: new QueryArguments(
+                    new QueryArgument<LongGraphType>
+                    {
+                        Name = "startIndex"
+                    },
+                    new QueryArgument<LongGraphType>
+                    {
+                        Name = "endIndex"
+                    },
+                    new QueryArgument<IntGraphType>
+                    {
+                        Name = "repeatCount",
+                        DefaultValue = 1
+                    }
+                ),
```

Comment:
Could you add a deprecation message? I think this query should be not used by everyone. Of course, it is useless but it will hide this query in the GraphQL Playground at least.